package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "catberry-handlebars",
 	"description": "Catberry adapter for Handlebars template engine",
-	"version": "2.0.1",
+	"version": "2.1.0",
```

Comment:
This is 3.0.0 definitely. We break backward compatibility using Catberry 8.0 as a peer dependency and we have to do that because of new service locator.